notebooks/correlation_vs_overlap.ipynb

```diff
@@ -0,0 +1,1735 @@
+{
```

Comment:
<p>Could be interesting to visualize this distribution as well (unless you did that somewhere else and I'm forgetting)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Good idea! The histogram didn't want to run (unsurprising I guess since the list is 750k long), but I added a table showing how many DOIs have show up in each bin</p>